truss/remote/remote_factory.py

```diff
@@ -0,0 +1,87 @@
+import configparser
+import inspect
+from pathlib import Path
+from typing import Dict, Type
+
+from truss.remote.baseten import BasetenRemote
+from truss.remote.truss_remote import TrussRemote
+
+
+class RemoteFactory:
+    """
+    A factory for instantiating a TrussRemote from a .trussrc file and a user-specified service name
+    """
+
+    REGISTRY: Dict[str, Type[TrussRemote]] = {"baseten": BasetenRemote}
+
+    @staticmethod
+    def load_service(service_name: str) -> Dict:
+        """
+        Load and validate a service from the .trussrc file
+        """
+        config_path = Path("~/.trussrc").expanduser()
+
+        if not config_path.exists():
+            raise FileNotFoundError(f"No .trussrc file found at {config_path}")
```

Comment:
when does this get created?

Reply:
(as per offline convo) this will be covered in a follow-up pr that builds an interactive cli. it'll def be supported before we launch `truss push`.